### PR TITLE
Update the Harvester cloud provider document

### PR DESCRIPTION
**Pull request by yaocw2020**

### CODE REVIEW

File: docs/rancher/cloud-provider.md [hunk 1]
```diff
@@ -30,61 +28,58 @@ In this page we will learn:
...
+
+| Annotation Key | Value Type | Required | Description |
 |:---|:---|:---|:---|
 | `cloudprovider.harvesterhci.io/healthcheck-port` | string | true | Specifies the port. The prober will access the address composed of the backend server IP and the port.
 | `cloudprovider.harvesterhci.io/healthcheck-success-threshold` | string | false | Specifies the health check success threshold. The default value is 1. If the number of times the prober continuously detects an address successfully reaches the success threshold, then the backend server can start to forward traffic.
 | `cloudprovider.harvesterhci.io/healthcheck-failure-threshold` | string | false | Specifies the success and failure threshold. The default value is 3. The backend server will stop forwarding traffic if the number of health check failures reaches the failure threshold.
 | `cloudprovider.harvesterhci.io/healthcheck-periodseconds` | string | false |  Specifies the health check period. The default value is 5 seconds.
 | `cloudprovider.harvesterhci.io/healthcheck-timeoutseconds` | string | false | Specifies the timeout of every health check. The default value is 3 seconds.
```
Comment (nunix) [inline docs/rancher/cloud-provider.md:81]:
```suggestion
| Annotation Key | Value Type | Required | Description |
|:---|:---|:---|:---|
| `cloudprovider.harvesterhci.io/healthcheck-port` | string | true | Specifies the port. The prober will access the address composed of the backend server IP and the port.
| `cloudprovider.harvesterhci.io/healthcheck-success-threshold` | string | false | Specifies the health check success threshold. The default value is 1. The backend server will start forwarding traffic if the number of times the prober continuously detects an address successfully reaches the threshold.
| `cloudprovider.harvesterhci.io/healthcheck-failure-threshold` | string | false | Specifies the health check failure threshold. The default value is 3. The backend server will stop forwarding traffic if the number of health check failures reaches the threshold.
| `cloudprovider.harvesterhci.io/healthcheck-periodseconds` | string | false |  Specifies the health check period. The default value is 5 seconds.
| `cloudprovider.harvesterhci.io/healthcheck-timeoutseconds` | string | false | Specifies the timeout of every health check. The default value is 3 seconds.
```

File: docs/rancher/cloud-provider.md [hunk 2]
```diff
@@ -30,61 +28,58 @@ In this page we will learn:
...
+
+!!!note
+    The health check port should be a `nodeport`. We will change it as the service port in the future release. Please check [this issue](https://github.com/harvester/harvester/issues/1697) for details.
```
Comment (nunix) [inline docs/rancher/cloud-provider.md:84]:
```suggestion
    The health check port is set as a `nodeport`. There's [a feature request](https://github.com/harvester/harvester/issues/1697) to change it to a service port and which will be implemented in a future release.
```

**Review by nunix — COMMENTED** [review]
Rephrased the options' description and the notes

File: docs/rancher/cloud-provider.md (hunk 2 above)
Comment (yaocw2020) [reply docs/rancher/cloud-provider.md:84]: The note is to tell users that a valid health check port should be a nodeport.

Comment (nunix) [reply docs/rancher/cloud-provider.md:84]: thank you, I updated my suggestion accordingly. Let me know if it makes more sense now.

**Review by nunix — APPROVED** [review]

**Review by guangbochen — APPROVED** [review]

**Pull request merged**